src/components/trade/TradesTable/index.tsx

```diff
@@ -0,0 +1,176 @@
+import { Order } from 'api/operator'
+import styled from 'styled-components'
+import { RowWithCopyButton } from 'components/orders/RowWithCopyButton'
+import React from 'react'
+// import styled from 'styled-components'
+// import { media } from 'theme/styles/media'
+import { SimpleTable } from '../../common/SimpleTable'
+import TradesTableContext from './Context/TradesTableContext'
+import { DateDisplay } from 'components/orders/DateDisplay'
+import { faExchangeAlt } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { OrderSurplusDisplay } from 'components/orders/OrderSurplusDisplay'
+import { calculatePrice, formatSmart, invertPrice, safeTokenName, TokenErc20 } from '@gnosis.pm/dex-js'
+import {
+  HIGH_PRECISION_DECIMALS,
+  HIGH_PRECISION_SMALL_LIMIT,
+  NO_ADJUSTMENT_NEEDED_PRECISION,
+} from 'apps/explorer/const'
+import BigNumber from 'bignumber.js'
+
+const Icon = styled(FontAwesomeIcon)`
```

Comment:
I see you are re-using the same code from https://github.com/gnosis/gp-ui/blob/534de13ce79703d081e753e0052c97048c9a1815/src/components/orders/OrderPriceDisplay/index.tsx#L20

Great idea!

In that case, extract this as a component to re-use it on both :)